.vitepress/config.mjs

```diff
@@ -43,8 +43,16 @@ export default defineConfig({
       // other locale specific properties...
       themeConfig: {
         sidebar: getSidebar.sidebar("en"),
+
         editLink: {
-          pattern: "https://github.com/PX4/PX4-user_guide/edit/main/:path",
+          pattern: ({ filePath, frontmatter }) => {
```

Comment:
This allows us to specify `newEditLink` in page metadata to fix up the edit link for a page that includes most of its data from another page.